### PR TITLE
Get valuta and booking date

### PR DESCRIPTION
**Pull request by MrCrankHank**
Parse the valuta and booking date from the response and add setters and getters to the model.

Confirmed working with the DKB and the ING-Diba.

### CODE REVIEW

File: lib/Fhp/Model/StatementOfAccount/Transaction.php [hunk 1]
```diff
@@ -54,25 +59,49 @@ class Transaction
...
+    {
+        return $this->bookingDate;
+    }
+
+    /**
      * Get date
      *
      * @return \DateTime|null
      */
-    public function getDate()
```
Comment (mschindler83) [inline lib/Fhp/Model/StatementOfAccount/Transaction.php]: Please do not delete the method "getDate".
As it is public, someone else could use it already in his code.
Better would be to return the booking date on "getDate" and just mark the method as deprecated.
So people have time to change their code to use one of the new methods.
Thanks :)

File: lib/Fhp/Parser/MT940.php [hunk 2]
```diff
@@ -91,8 +91,6 @@ protected function parseToArray()
                     $transaction = substr($day[$i], 3);
                     $description = substr($day[$i + 1], 3);
 
-                    //$transactionDate = $this->getDate(substr($transaction, 0, 6));
```
Comment (mschindler83) [inline lib/Fhp/Parser/MT940.php]: You can remove this line. It's no longer needed.

**Comment by mschindler83** [comment]
Hey @MrCrankHank 
Nice PR! I just added some comments. Maybe you can have a look if you have time.

It would be also great if you could update the COMPATIBILITY.md file if you had the chance to test the library with different banks. This would be a huge help!

Have a great day!

File: lib/Fhp/Parser/MT940.php (hunk 2 above)
Comment (MrCrankHank) [inline lib/Fhp/Parser/MT940.php]: Not sure what you mean here. I deleted the comment?

**Comment by MrCrankHank** [comment]
Hi!

I readded the getDate() method and updated the compatiblity file.

Unfortunatly I only have access to the DKB and the ING-Diba, so I can't provide any further testing.

Have a great day two.

**Comment by mschindler83** [comment]
Perfect!
Merging PR and closing issue.
Thanks !

**Pull request merged**